src/utils/CategoryUtils/CategoryUtils.ts

```diff
@@ -0,0 +1,113 @@
+import {CustomIconNames} from '../../components';
+import {Constants} from '../../constants';
+import {CategoryInterface} from '../../models';
+import {FeaturedCategories, MoreCategories} from './CategoryUtils.types';
+
+const featuredCategories = (categories: CategoryInterface[]) => {
+  const topFeaturedItems = [
+    'restaurants',
+    'shopping',
+    'nightlife',
+    'active',
+    'beautysvc',
+    'homeservices',
+  ];
+
+  const moreItemArray = [
+    {
+      alias: 'more',
+      title: 'More',
```

Comment:
Localisation

Reply:
All the other featured categories are coming only in english, keeping this one item in arabic i think doesnot look good so i havent changed it.